cmd/livepeer/livepeer.go

```diff
@@ -650,14 +664,6 @@ func main() {
 		} else if len(orchURLs) > 0 {
 			n.OrchestratorPool = discovery.NewOrchestratorPool(bcast, orchURLs)
 		} else if *network != "offchain" {
```

Comment:
I think we can move this codeblock up in its entirety ? and overwrite `n.ORchestratorPool` with `n.OrchestratorPool = discovery.NewOrchestratorPool(bcast, orchURLs)` when we have `orchURLs > 0` which will always we be the case when we run offchain. 

In the case we use a webhook , this will also be overwritten if the option is defined. 

Might have looked over something here though

Reply:
Fixed in 8633ae4